portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeStandardHead.vm

```diff
@@ -59,10 +59,12 @@
                 },
                 "portalCDNQuery" : "$!{portalCDNQuery}",
                 "sakaiSkin" : "$!{pageSkin}",
-                "userTheme" : "$!{userTheme}"
-                #if ($themesAutoDetectDark)
-                ,"userThemeAutoDetectDark" : true
-                #end
+                "userTheme" : "$!{userTheme}",
+                #if($themesAutoDetectDark)"userThemeAutoDetectDark" : true,
```

Comment:
move the end to the line, 

#if($themesAutoDetectDark)"userThemeAutoDetectDark" : true,#end